config/configretry/backoff.go

```diff
@@ -0,0 +1,66 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package configretry
+
+import (
+	"errors"
+	"time"
+
+	"github.com/cenkalti/backoff/v4"
+)
+
+// NewDefaultBackOffSettings returns the default settings for RetrySettings.
+func NewDefaultBackOffSettings() BackOffSettings {
+	return BackOffSettings{
+		Enabled:             true,
+		InitialInterval:     5 * time.Second,
+		RandomizationFactor: backoff.DefaultRandomizationFactor,
+		Multiplier:          backoff.DefaultMultiplier,
+		MaxInterval:         30 * time.Second,
+		MaxElapsedTime:      5 * time.Minute,
+	}
+}
+
+// BackOffSettings defines configuration for retrying batches in case of export failure.
+// The current supported strategy is exponential backoff.
+type BackOffSettings struct {
```

Comment:
I think we should use `BackOffConfig` since it's user configuration as proposed in
https://github.com/open-telemetry/opentelemetry-collector/issues/6767

Reply:
Are we committed to do that issue?

Reply:
I believe, yes. I didn't hear any objections. I just didn't make any changes specifically to address that it the exporterhelper knowing that it can change. I use the `Config` suffix for the new `WithQueue` option in https://github.com/open-telemetry/opentelemetry-collector/pull/8853

Reply:
Done.